src/css/about.css

```diff
@@ -0,0 +1,194 @@
+
+.about-card-container {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: center;
+    gap: 16px;
+}
+
+.about-header {
+    font-family: var(--primary-font-family);
+    font-weight: var(--bold-font-weight);
+    font-size: 32px;
+    line-height: 1.125;
+    letter-spacing: -0.02em;
+    color: var(--black-color);
+}
+
+.about-marker {
+    position: relative;
+}
+
+.about-marker::after {
+    content: "";
+    position: absolute;
+    display: block;
+    top: 29px;
+    right: -2px;
+    width: 41px;
+    height: 12px;
+    border-radius: 8px;
+    background-color: #BEE8CC;
+    z-index: -1;
+}
+
+.about-paragraph {
+    font-family: var(--primary-font-family);
+    font-weight: var(--regular-font-weight);
+    font-size: 14px;
+    line-height: 1.714;
+    text-align: center;
+    color:  #12141780;
+}
+
+.about-list-container {
+    display: flex;
+    flex-direction: column;
+    width: 100%;
+    margin-top: 40px;
+    gap: 20px;
+}
+
+.about-list {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: center;
+    gap: 20px;
+}
+
+.about-list-item {
+    display: flex;
+    flex-direction: column;
+    padding: 32px;
+    background-color: var(--gray-color);
+    border-radius: 16px;
+}
+
+.about-list-item-container {
+    display: flex;
+    flex-direction: column;
+    align-items: flex-start;
+    justify-content: center;
+    gap: 16px;
+}
+
+.about-list-header {
+    font-family: var(--primary-font-family);
+    font-weight: var(--bold-font-weight);
+    font-size: 28px;
+    line-height: 1.142;
+    letter-spacing: -0.02em;
+    color: var(--green-color);
+}
+
+.about-list-paragraph {
+    font-family: var(--primary-font-family);
+    font-weight: var(--bold-font-weight);
+    font-size: 20px;
+    line-height: 1.2;
+    letter-spacing: -0.02em;
+    color: var(--black-color);
+}
+
+.about-picture {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: center;
+    width: 100%;
+    height: 313px;
+    border-radius: 16px;
+    background-color: var(--green-color);
+}
+
+/* Styles for Tablet devices */
+@media only screen and (min-width: 768px) {
+    .about-card-container {
+        padding: 0 23px;
+    }
+
+    .about-heder {
+        font-size: 40px;
+        line-height: 1.2;
+        letter-spacing: -0.02em;
+    }
+    
+    .about-marker::after {
```

Comment:
look how the element is positioned, different views (mobile, tablet, desktop), different position